server/chat-jsx.tsx

```diff
@@ -36,3 +37,16 @@ export type VNode = preact.VNode;
 
 export const h = preact.h;
 export const Fragment = preact.Fragment;
+
+export class FormatText extends preact.Component<{isTrusted?: boolean, replaceLinebreaks?: boolean}> {
+	render() {
+		const child = this.props.children;
+		if (typeof child === 'string') {
+			return <span dangerouslySetInnerHTML={
+				{__html: Chat.formatText(child, this.props.isTrusted, this.props.replaceLinebreaks)}
+			} />;
+		}
+		if (preact.isValidElement(child)) return child;
```

Comment:
Why isn't this a throw? This seems like this should throw.

Reply:
Like if you use `FormatText` and you don't use it to format your text, that clearly seems like a bug.

Reply:
That’s fair

Reply:
did this btw @Zarel